pycytominer/feature_select.py

```diff
@@ -38,44 +39,62 @@ def feature_select(
     noise_removal_perturb_groups=None,
     noise_removal_stdev_cutoff=None,
 ):
-    """
-    Performs feature selection based on the given operation
+    """Performs feature selection based on the given operation.
+
+    Parameters
+    ----------
+    profiles : pandas.core.frame.DataFrame or file
+        DataFrame or file of profiles.
+    features : list
+        A list of strings corresponding to feature measurement column names in the
+        `profiles` DataFrame. All features listed must be found in `profiles`.
+        Defaults to "infer". If "infer", then assume cell painting features are those
+        prefixed with "Cells", "Nuclei", or "Cytoplasm".
+    image_features: bool, default False
+        Whether the profiles contain image features.
+    samples : list or str, default "all"
+        Samples to provide operation on.
+    operation: list of str or str, default "variance_threshold
+        Operations to perform on the input profiles.
+    output_file : str, optional
+        If provided, will write annotated profiles to file. If not specified, will
+        return the normalized profiles as output. We recommend that this output file be
+        suffixed with "_normalized_variable_selected.csv".
+    na_cutoff : float, default 0.05
+        Proportion of missing values in a column to tolerate before removing.
+    corr_threshold : float, default 0.1
+        Value between (0, 1) to exclude features above if any two features are correlated above this threshold.
+    corr_method : str, default "pearson"
+        Correlation type to compute. Allowed methods are "spearman", "kendall" and "pearson".
+    freq_cut : float, default 0.1
+        Ratio (2nd most common feature val / most common).
+    unique_cut: float, default 0.1
+        Ratio (num unique features / num samples).
```

Comment:
@flohu noted as a minor point in #167 that there are discrepancies between the default values in the function and those listed in the docstring.

@niranjchandrasekaran, since you're editing this docstring, you might as well fix the discrepancy